ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcAcidRowBatchReader.java

```diff
@@ -2039,4 +2189,29 @@ private static IntegerColumnStatistics deserializeIntColumnStatistics(List<OrcPr
         deserializeIntColumnStatistics(colStats, OrcRecordUpdater.ROW_ID + 1);
     return getKeyInterval(columnStatsArray);
   }
+
+  private static class RowIsDeletedColumnVector extends LongColumnVector {
+    void clear() {
+      Arrays.fill(vector, 0);
+    }
+
+    void setAll(boolean deleted) {
+      vector[0] = deleted ? 1 : 0;
```

Comment:
seems like a duplication of LongColumnVector's fill(long value) method

Reply:
Changed the implementation to call `LongColumnVector.fill`